nomad/structs/structs.go

```diff
@@ -127,10 +127,13 @@ const (
 	ACLBindingRulesDeleteRequestType             MessageType = 58
 	NodePoolUpsertRequestType                    MessageType = 59
 	NodePoolDeleteRequestType                    MessageType = 60
+	JobVersionTagRequestType                     MessageType = 61
```

Comment:
Phil has this message ID reserved in another PR, so I'm trying to avoid creating a conflict for him here :grin: